packages/server/src/fhir/search.test.ts

```diff
@@ -4027,6 +4028,41 @@ describe('FHIR Search', () => {
             'Observation',
           ]);
         }));
+
+      test('Cursor pagination', () =>
+        withTestContext(async () => {
+          const tasks: Task[] = [];
+          let prevLastUpdated = '';
+          for (let i = 0; i < 10; i++) {
+            const task = await repo.createResource<Task>({
+              resourceType: 'Task',
+              status: 'accepted',
+              intent: 'order',
+              code: { text: 'cursor_test' },
+            });
+            expect(task.meta?.lastUpdated).toBeDefined();
+            expect(task.meta?.lastUpdated).not.toEqual(prevLastUpdated);
+            tasks.push(task);
+            prevLastUpdated = task.meta?.lastUpdated as string;
+            await sleep(1);
```

Comment:
Can you use `jest.useFakeTimers` and `jest.advanceTimersByTime ` instead?

Reply:
After using `lastUpdated` and `id` together, the sleep and timers are unnecessary.